data/human/culture conversations.txt

```diff
@@ -344,11 +344,22 @@ mission "Forest Fire"
 mission "Local Politics"
 	minor
 	source
-		government "Free Worlds"
 		attributes "frontier"
-		not attributes "station"
+		not attributes "station" "urban"
 	to offer
-		random < 1
+		or
+			and
+				random < 5
+				or
+					has "flagship planet attribute: south pirate"
+					and
+						not "event: war begins"
+						has "flagship planet attribute: south"
+			and
+				random < 1
+				has "event: war begins"
+				not "event: fw suppressed Greenrock"
+				has "flagship planet attribute: south"
```

Comment:
Removing redundant attribute checks.
```suggestion
			and
				random < 5
				or
					has "flagship planet attribute: south pirate"
					not "event: war begins"
			and
				random < 1
				has "event: war begins"
				not "event: fw suppressed Greenrock"
```
"Any southern frontier non-station, non-urban world (from the source filter) where either the war hasn't begun, the world has the south pirate attribute, or the war has begun but the FW haven't handled piracy." Is that the intended offer location? Note that we do not remove the "south pirate" attribute from pirate planets after they change their government, so this actually still allows pirate worlds that have changed sides to the FW to get this mission to offer. I imagine that if we don't want this to offer, then we should be removing those attributes where appropriate from events.txt.